.github/workflows/lint.yaml

```diff
@@ -6,32 +6,36 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - name: Install Go
-        uses: actions/setup-go@v2
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Lint
-        run: |
-          docker run --rm -v `pwd`:/go/src/prj -w /go/src/prj \
-            golangci/golangci-lint golangci-lint run \
-                -v \
-                --max-same-issues 10 \
-                --disable-all \
-                -E asciicheck \
-                -E deadcode \
-                -E errcheck \
-                -E forcetypeassert \
-                -E gocritic \
-                -E gofmt \
-                -E goimports \
-                -E gosimple \
-                -E govet \
-                -E ineffassign \
-                -E misspell \
-                -E revive \
-                -E staticcheck \
-                -E structcheck \
-                -E typecheck \
-                -E unused \
-                -E varcheck
+        uses: golangci/golangci-lint-action@v2
+        with:
+          # version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
+          version: latest
```

Comment:
Not sure which version was installed before.

"latest" obviously has the risk that newer versions will flag problems in the code base that weren't found before. I still prefer that over manually bumping the version because in practice we probably woudn't remember to do that...

Reply:
agree